src/content/docs/en/guides/data-fetching.mdx

```diff
@@ -49,7 +49,7 @@ The `fetch()` function is also globally available to any [framework components](
 import type { FunctionalComponent } from 'preact';
 
 const data = await fetch('https://example.com/movies.json').then((response) =>
-  response.json()
+  response.json();
```

Comment:
If you update by referencing the changes in this PR, please include the changes in [this PR](https://github.com/withastro/docs/pull/11205/files) as well! Thank you!